benchmark/cublas/main.cpp

```diff
@@ -0,0 +1,38 @@
+#include "cublas_v2.h"
+#include "cuda_runtime_api.h"
+#include "utils.hpp"
+#include <common/cli_device_selector.hpp>
+#include <common/print_queue_information.hpp>
+
+// Create a shared pointer to a sycl blas Handle, so that we don't keep
+// reconstructing it each time (which is slow). Although this won't be
+// cleaned up if RunSpecifiedBenchmarks exits badly, that's okay, as those
+// are presumably exceptional circumstances.
+std::unique_ptr<utils::cli_device_selector> cdsp;
+void free_device_selector() { cdsp.reset(); }
+
+int main(int argc, char** argv) {
+  // Read the command-line arguments
+  auto args = blas_benchmark::utils::parse_args(argc, argv);
+
+  // Initialize googlebench
+  benchmark::Initialize(&argc, argv);
+
+  // Create a sycl blas sb_handle from the queue
+  cublasHandle_t cublas_handle = NULL;
+  cublasCreate(&cublas_handle);
```

Comment:
```suggestion
  CUBLAS_CHECK(cublasCreate(&cublas_handle));
```

I would be good idea to check for errors the return values of `cublas` and `cuda` function calls.

Reply:
I am using them when I write the benchmark to understand if everything works fine, then I remove them. Should I keep them all?

Reply:
Yes, it is advisable to keep them in order to be aware of possible runtime errors, e.g., `cublas_handler` creation problems due to system configurations, memory allocation errors, and so on.